defaults/main/main.yml

```diff
@@ -0,0 +1,215 @@
+
+# Because problems with some pip versions, we added this option to choose what version the role should install.
+# See https://airflow.apache.org/docs/apache-airflow/stable/installation.html#installation-tools
+airflow_pip_version: 20.2.4
+
+# Airflow url to download the correct version
+airflow_constraint_url: "https://raw.githubusercontent.com/apache/airflow/constraints-{{ airflow_version }}/constraints-{{ airflow_python_version }}.txt"
+
+# Available extra packages:
+# https://airflow.apache.org/docs/apache-airflow/stable/extra-packages-ref.html
+# List should follow Ansible's YAML basics:
+# https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics
+airflow_bundle_package: # all|all_dbs|devel|devel_hadoop|devel_all|devel_ci See bundle extras section 
+# airflow_extra_packages:
+  # [Apache]
+  # - apache.atlas
+  # - apache.beam
+  # - apache.cassandra
+  # - apache.druid
+  # - apache.hdfs
+  # - apache.hive
+  # - apache.kylin
+  # - apache.livy
+  # - apache.pig
+  # - apache.pinot
+  # - apache.spark
+  # - apache.sqoop
+  # - apache.webhdfs
+  # [Services]
+  # - amazon
+  # - microsoft.azure
+  # - cloudant
+  # - databricks
+  # - datadog
+  # - dask
+  # - dingding
+  # - discord
+  # - facebook
+  # - google
+  # - github_enterprise
+  # - google_auth
+  # - hashicorp
+  # - jira
+  # - opsgenie
+  # - pagerduty
+  # - plexus
+  # - qubole
+  # - salesforce
+  # - sendgrid
+  # - segment
+  # - sentry
+  # - slack
+  # - snowflake
+  # - telegram
+  # - vertica
+  # - yandex
+  # - zendesk
+  # [Software]
+  # - async
+  # - celery
+  # - cncf.kubernetes
+  # - docker
+  # - elasticsearch
+  # - exasol
+  # - jenkins
+  # - ldap
+  # - mongo
+  # - microsoft.mssql
+  # - mysql
+  # - odbc
+  # - openfaas
+  # - oracle
+  # - postgres
+  # - password
+  # - presto
+  # - rabbitmq
+  # - redis
+  # - samba
+  # - singularity
+  # - statsd
+  # - tableau
+  # - virtualenv
+  # [Others]
+  # - cgroups
+  # - ftp
+  # - grpc
+  # - http
+  # - imap
+  # - jdbc
+  # - kerberos
+  # - papermill
+  # - sftp
+  # - sqlite
+  # - ssh
+  # - microsoft.winrm
+
+airflow_default_required_libs:
+  - python3-dev
+  - python3-pip
+  - python3-setuptools
+  - python-pip
+  # - acl
+# Airflow 2.0 system level requirements https://airflow.apache.org/docs/apache-airflow/stable/installation.html#system-dependencies
+  - freetds-bin
+  - krb5-user
+  - ldap-utils
+  - libffi6
+  - libsasl2-2
+  - libsasl2-modules
+  - libssl1.1
+  - locales
+  - lsb-release
+  - sasl2-bin
+  - sqlite3
+  - unixodbc
+
+airflow_additional_required_libs: []
+
+airflow_required_libs: "{{ airflow_default_required_libs + airflow_additional_required_libs }}"
+
+# airflow_required_python_packages:
+  # - { name: SQLAlchemy, version: 1.3.23 } # v1.4 Breaks airflow installation
+#   Version is not mandatory
+#   - {name: pyasn1, version: 0.4.4}
+
+# Owner
+airflow_user: airflow
+airflow_group: airflow
+
+# Admin user
+airflow_admin_users:
```

Comment:
We might want to leave this empty (add it on molecule variables for testing).
Doing it this way, there will never be a default admin user on playbooks. Users will have to define them manually, removing the risk of having by accident a default admin/admin user/pass :)

Either way, LGTM!

Reply:
Done! and warn about this admin values in README.md